application/models/model_login.php

```diff
@@ -4,6 +4,19 @@ class Model_Login extends Model
 {
 	public function get_data()
 	{	
-	
+		$bd = new Database();
+
+		$name = $_POST['name'];
```

Comment:
check if post[name] and post[password] are gotten

Reply:
everything is working